executor/containerdexecutor/executor.go

```diff
@@ -47,6 +51,13 @@ func (w containerdExecutor) Exec(ctx context.Context, meta executor.Meta, root c
 		return err
 	}
 
+	hostNetworkEnabled := false
```

Comment:
This should be configurable

Reply:
Do you mean, user input if CNI networking should be used or not?

Currently, It tries to setup NetworkInterface with CNI, on failure uses the host networking.

Reply:
Yes, e.g. `buildkitd --containerd-worker-net=(host|cni)`

Reply:
got it. Will add the option